src/utils/perform_search.py

```diff
@@ -18,8 +19,9 @@ def perform_search(sketch_path, db_name, max_results=10):
     print('search done in', time.time() - start_time)
     if response.status_code == 200:
         resp_json = response.json()
-        # Convert Refseq IDs into KBase IDs
+        # Convert Refseq IDs into KBase IDs (only does these if we are in the refseq namespace)
```

Comment:
is there somewhere where we check which namespace we're in before doing "map_strains"? I dont see this

Reply:
right now no, but the way they work it shouldn't fail, but I'll add that check for good measure